### ISSUE TITLE
Idea; search or add option - faster add new todo items feature

### ISSUE THREAD

**Issue by clach04** [mpcjanssen/simpletask-android#1199]
Have a search box that live searches, then have an option to add the content of the search as a new line item

I had this idea for a feature, and then found out https://github.com/openintents/shoppinglist implements something like it. OI shopping list shares "lists" in common with simpletask/todo.txt but that's about it.

The OI list feature is implemented as live search, when pressing enter/return the value is added to the list. Right now that would conflict with the current SimpleTask behavior where pressing enter performs the search. I'll attach screen shots (the Google Play store screen shots don;t show this, in fact they don;t reflect what it really looks like at all)

An alternative is to pop-up buttons to the right hand side of the search field. E.g. "Search" and "Add", that way if the search doesn't get good hits, a single button press would add the new line item.

**Comment by clach04** [comment]
![Screenshot_20220910-145021_OI Shopping List](https://user-images.githubusercontent.com/424032/189503213-8188402c-46f3-4877-a3e5-3c0f145e800c.jpg)
![Screenshot_20220910-145128_OI Shopping List](https://user-images.githubusercontent.com/424032/189503215-e7daf627-ae04-48a4-8c84-952b44aae651.jpg)

**Comment by mpcjanssen** [comment]
Good idea. Probably not super easy to implenent considerign how difficult it is to do stuff with the standard Android search widget.
